examples/nextjs/pages/agent.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  LiveKitRoom,
+  useToken,
+  setLogLevel,
+  useVoiceAssistant,
+  AgentBarVisualizer,
+  ControlBar,
+  RoomAudioRenderer,
+  // VoiceAssistantControlBar,
+} from '@livekit/components-react';
+import type { NextPage } from 'next';
+import { generateRandomUserId } from '../lib/helper';
+import { useEffect, useMemo } from 'react';
+
+const MinimalExample: NextPage = () => {
+  const params = typeof window !== 'undefined' ? new URLSearchParams(location.search) : null;
+  const roomName = useMemo(
+    () => params?.get('room') ?? 'test-room-' + Math.random().toFixed(5),
+    [],
+  );
+  setLogLevel('info', { liveKitClientLogLevel: 'debug' });
+
+  const tokenOptions = useMemo(() => {
+    const userId = params?.get('user') ?? generateRandomUserId();
+    return {
+      userInfo: {
+        identity: userId,
+        name: userId,
+      },
+    };
+  }, []);
+
+  const token = useToken(process.env.NEXT_PUBLIC_LK_TOKEN_ENDPOINT, roomName, tokenOptions);
+
+  function AgentTile() {
+    const { state, audioTrack } = useVoiceAssistant();
```

Comment:
@lukasIO I'm assuming that we need to make AgentTile its own component because `useVoiceAssistant()` has to run inside of a `LiveKitRoom` context? If so - it seems a little clunky, no? Would it be possible for `AgentBarVisualizer` to just do this internally somehow? it would be nice if you could just do this:

```
<LiveKitRoom ...>
  <VoiceAssistantBarVisualizer />
  <VoiceAssistantControlBar />
</LiveKitRoom>
```